kytos/utils/napps.py

```diff
@@ -486,11 +493,11 @@ def build_napp_package(napp_name):
                 if re.search(ignored+"$", filename):
                     files.remove(filename)
                     break
-
         # Create the '.napp' package
         napp_file = tarfile.open(napp_name + '.napp', 'x:xz')
         for local_f in files:
-            napp_file.add(local_f)
+            # Add relative paths instead of absolute paths
+            napp_file.add(local_f.replace(path+'/', ''))
```

Comment:
Use of `Path` objects and their methods can make this code more readable.
(eg.: [Path.relative_to](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to))